packages/realm/src/platform/node/fs.ts

```diff
@@ -19,8 +19,8 @@
 import { existsSync, mkdirSync, readdirSync, rmSync, unlinkSync } from "node:fs";
 import { dirname, isAbsolute, join } from "node:path";
 
-import { inject } from "../platform/file-system";
-import { extendDebug } from "../debug";
+import { inject } from "../file-system";
+import { extendDebug } from "../../debug";
```

Comment:
We should consider defining a virtual path `REALM/` or `src/` so that imports look the same regardless of where you are in the directory structure. It is really unfortunate that when you move a file around, you need to modify its own imports, not just the imports _of_ it. This is pretty common in C++ projects to always use "fully qualified" paths in #includes, so you use `#include "realm/object-store/object.hpp"` rather than `#include "./object.hpp"` if you are in `realm/object-store/blah.hpp`.

Reply:
I think that's a good idea, but I'll leave it for a future PR 👍 Ideally I would also like to find a way to enforce that we import through the `internal` module instead of relative paths (for all other files than the platform specific).